frontend/src/pages/product-search/ProductSearch.jsx

```diff
@@ -127,64 +127,77 @@ function ProductSearch() {
 	return (
         <div>
             <Loading isLoading={isLoading} percent={loadingPercent} size={200} strokeWidth={12} />
-            <div className="fixed-buttons-container">
+
+            {productData.length > 0 && numProductsSelected > 2 && <div className="fixed-buttons-container">
                 <div className="fixed-buttons">
-                    <button
+                    {/* <button
                         onClick={() => navigate(routes.home)}
                         className="primary-button-inverted primary-button-size"
-                    >Cancel</button>
+                    >Cancel</button> */}
                     <button
-                        disabled={isNextButtonDisabled}
+                        // disabled={isNextButtonDisabled}
```

Comment:
We no longer have any disabled functionality on the button? Doesn't this allow users to select infinitely many products now?

Reply:
I didn't know that the disabled functionality limited amount of products to select? Does it? Either way idk I think this is fine